### PR TITLE
Event translations

### PR DESCRIPTION
**Pull request by Aliiiu**
# Event Translations

This PR resolves translations regarding events

## Test
- event and event details page translation
- Preview missing dates on each event page
- missing locations from event card but present in event details page

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2962

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
or the Chinese and Korean versions, let's split the header text into two lines. The current spacing below the header feels unbalanced.
image

**Comment by Aliiiu** [comment]
> Translating to Korean (Ko) gave an entirely different outcome , the past event and upcoming events aren't displaying on the screen again
> 
> image

@Softdev1 Test it again
https://iq-wiki-git-event-translations-prediqt.vercel.app/events/

**Review by Simplypheyie96 — APPROVED** [review]

**Review by Softdev1 — APPROVED** [review]

**Review by Yomzy2223 — COMMENTED** [review]
Looks good, but why not follow a particular naming convention for the keys, for consistency and readability sake? I can see some in camel case, and some pascal.

**Comment by Aliiiu** [comment]
> Looks good, but why not follow a particular naming convention for the keys, for consistency and readability sake? I can see some in camel case, and some pascal.


Can you drop screenshot of examples so I can work on them

**Review by Yomzy2223 — APPROVED** [review]

File: public/locales/ko/event.json [hunk 1]
```diff
@@ -6,5 +6,81 @@
...
+  "clearAll": "모두 지우기",
+  "date": "날짜",
+  "Next Week": "다음 주",
+  "Next Month": "다음 달",
+  "Custom Range": "사용자 정의 범위",
+  "location": "위치",
+  "Asia": "아시아",
+  "Africa": "아프리카",
+  "Europe": "유럽",
+  "North America": "북아메리카",
```
Comment (Adebesin-Cell) [inline public/locales/ko/event.json:45]: Can this be made camelCase as well?

Comment (Aliiiu) [reply public/locales/ko/event.json:45]: They are static data that are used for filtering. When creating an event we don't have options for regions in different languages. Same with all other filtering options and interests.

Comment (Adebesin-Cell) [reply public/locales/ko/event.json:45]: Oh fair enough!

**Review by Adebesin-Cell — APPROVED** [review]

File: public/locales/ko/event.json (hunk 1 above)
Comment (Aliiiu) [reply public/locales/ko/event.json:45]: image
image

Comment (Aliiiu) [reply public/locales/ko/event.json:45]: image

**Pull request merged**